features/font-variant-numeric.yml

```
@@ -0,0 +1,16 @@
+name: Font variants for numbers
+description: The `font-variant-numeric` CSS property sets how numeric characters are displayed. For example, you can specify that columns of numbers should be aligned, or that zeroes should have a slash.
```

I try to avoid "specify" wherever possible, as being spec speak and not very developer oriented. What do you think of this?

```suggestion
description: The `font-variant-numeric` CSS property sets how numeric characters are displayed. For example, you can align columns of numbers or use zeroes that have a slash.
```

---

I like this better. Thanks!